core/src/main/java/com/linecorp/armeria/client/HttpClientBuilder.java

```diff
@@ -45,6 +45,16 @@ public HttpClientBuilder(String uri) {
         this(URI.create(requireNonNull(uri, "uri")));
     }
 
+    /**
+     * Creates a new instance.
+     *
+     * @throws IllegalArgumentException if the {@code scheme} is not one of the fields
+     *                                  in {@link SessionProtocol}
+     */
+    public HttpClientBuilder(String scheme, Endpoint endpoint) {
```

Comment:
`String scheme` should be replaced with `SessionProtocol protocol`, because the serialization format is always `none` for `HttpClient`.